tests/logictest/complete.py

```diff
@@ -121,9 +137,14 @@ def gen_suite_from_sql(sqls, dest_file):
                         continue
                     rows.append(str(v))
                 results.append(rows)
-            statements.append(
-                f"statement query {options}\n{sql}\n\n----\n{format_result(results)}\n"
-            )
+            if len(results) > 0:
+                statements.append(
+                    f"statement query {options}\n{sql}\n\n----\n{format_result(results)}\n"
+                )
+            else:
+                statements.append(
+                    f"statement ok\n{sql}\n"
+                )
         except mysql.connector.Error as err:
             statements.append(f"statement ok\n{sql}\n\n")
```

Comment:
better change to `statements.append(f"statement error {err.errno}\n{sql}\n\n")`

Reply:
It's `No result set to fetch from.`

Reply:
> It's `No result set to fetch from.`

Fine, that is a statement ok